### PR TITLE
[Inji 388] : Error message in QR login

### PR DESCRIPTION
**Pull request by Alka1703**

### CODE REVIEW

File: shared/telemetry/TelemetryConstants.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ export const TelemetryConstants = {
     hardwareKeyStore:
       'Some security features will be unavailable as hardware key store is not available',
     activationCancelled: 'Activation Cancelled',
+    vcsAreTampered: 'VC are tampered',
```
Comment (swatigoel) [inline shared/telemetry/TelemetryConstants.js:33]: Why this change in this commit?

Comment (Alka1703) [reply shared/telemetry/TelemetryConstants.js]: Seems like this was a miss from some previous story and since it was an error event it went highly undetected.

**Review by swatigoel — APPROVED** [review]
